packages/osrm/osrm.test.ts

```diff
@@ -6,8 +6,8 @@ describe("OSRM returns responses", () => {
         await o
             .directions(
                 [
-                    [1.51886, 42.5063],
-                    [1.53789, 42.51007],
+                    [42.5063, 1.51886],
```

Comment:
Why is it lat lon in the test?

Reply:
I decided to have the API use Lat/Lon ordering. This was just the last missing Lon/Lat I forgot to change. Now thinking about it, more and more geospatial libs in the web realm are using Lon/Lat (pretty much only Leaflet is left, along with some OGC specifications like WMS). Let me open an issue so this will be addressed separately :)